server/re/include/irods_re_plugin.hpp

```diff
@@ -21,7 +20,15 @@
 #include <boost/any.hpp>
 #include <boost/algorithm/string.hpp>
 
-using logger = irods::experimental::log;
+#ifdef IRODS_ENABLE_SYSLOG
+    #include "irods_logger.hpp"
+
+    #define IRODS_HIDE_ON_CLIENT_SIDE(x) x
```

Comment:
IRODS_SERVER_ONLY ?

Reply:
that works too. im not a fan of either really, but `IRODS_SERVER_ONLY` is fewer characters.

if you read this two years from now, would you know what it meant just from reading it?

Reply:
i commented because it seemed to be a positive rather than a negative.   happens to be fewer characters...

does it need to start with IRODS?   let's talk in the morning.